fixtures/lightmaxx/cls-nano-cob.json

```
@@ -0,0 +1,208 @@
+      "https://www.youtube.com/watch?v=_Jfp0JCAOns"
+    ]
+  },
+  "physical": {
+    "dimensions": [720, 200, 110],
+    "weight": 5.7,
+    "power": 130,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [25, 25]
+    }
+  },
+  "matrix": {
+    "pixelCount": [4, 1, 1],
+    "pixelGroups": {
+      "All": "all"
+    }
+  },
+  "templateChannels": {
+    "Red $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    }
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "capability": {
+        "type": "ShutterStrobe",
+        "shutterEffect": "Strobe",
+        "speedStart": "slow",
+        "speedEnd": "fast"
+      }
+    },
+    "Color Presets": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 9],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [10, 99],
+          "type": "Intensity",
+          "brightness": "100%",
+          "comment": "Full ON"
+        },
+        {
+          "dmxRange": [100, 129],
+          "type": "Effect",
+          "effectPreset": "ColorJump",
+          "comment": "Color change erratic"
+        },
+        {
+          "dmxRange": [130, 159],
+          "type": "Effect",
+          "effectPreset": "ColorFade",
+          "comment": "Color change fade"
+        },
+        {
+          "dmxRange": [160, 189],
+          "type": "Effect",
+          "effectPreset": "ColorJump",
+          "comment": "Color change"
+        },
+        {
+          "dmxRange": [190, 219],
+          "type": "Effect",
+          "soundControlled": true,
+          "effectName": "Music Mode 1"
+        },
+        {
+          "dmxRange": [220, 249],
+          "type": "Effect",
+          "soundControlled": true,
+          "effectName": "Music Mode 2"
+        },
+        {
+          "dmxRange": [250, 255],
+          "type": "Effect",
+          "soundControlled": true,
+          "effectName": "Music Mode 3"
+        }
+      ]
+    },
+    "Program Speed": {
+      "capability": {
+        "type": "EffectSpeed",
+        "speedStart": "slow",
+        "speedEnd": "fast",
+        "comment": "Speed of Channel 6"
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "4-channel",
+      "shortName": "4ch",
+      "channels": [
+        "Dimmer",
+        {
+          "insert": "matrixChannels",
+          "repeatFor": ["All"],
+          "channelOrder": "perPixel",
+          "templateChannels": [
+            "Red $pixelKey",
+            "Green $pixelKey",
+            "Blue $pixelKey"
+          ]
+        }
+      ]
```

Instead of using an `insert` block, you can also reference the implicit matrix channels directly, which makes sense here:

```suggestion
      "channels": [
        "Dimmer",
        "Red All",
        "Green All",
        "Blue All"
      ]
```

---

Ahh, that's a great shorthand. Thanks!